setup.cfg

```diff
@@ -13,6 +13,9 @@ max-complexity = 15
 [tool:isort]
 profile = black
 
+[tool:pytest]
+asyncio_mode = auto
+
```

Comment:
Is it required?

Reply:
A will check on UTests for the async framework you already have. I'm wondering how it's implemented without asyncio_mode. Probably because of this also python 3.6 checks failed.